rest-service/manager_rest/flask_utils.py

```diff
@@ -64,13 +63,19 @@ def _get_postgres_db_uri(manager_ip, driver):
     )
     if config.instance.postgresql_ssl_enabled:
         conn_string += \
-            '?sslmode={sslmode}&' \
-            'sslcert={sslcert}&' \
-            'sslkey={sslkey}&' \
+            '?sslmode={sslmode}&'
+        ssl_mode = 'verify-ca'
+        if config.instance.postgresql_ssl_client_verification:
+            conn_string += \
+                'sslcert={sslcert}&' \
+                'sslkey={sslkey}&'.format(
+                    sslcert=config.instance.postgresql_ssl_cert_path,
+                    sslkey=config.instance.postgresql_ssl_key_path,
+                )
+            ssl_mode = 'verify-full'
+        conn_string += \
             'sslrootcert={sslrootcert}'.format(
-                sslmode='verify-full',
-                sslcert=config.instance.postgresql_ssl_cert_path,
-                sslkey=config.instance.postgresql_ssl_key_path,
+                sslmode=ssl_mode,
```

Comment:
this param isn't used in the format string

Reply:
outdated